### DISCUSSION TITLE
brew update —preinstall

### DISCUSSION THREAD

**Discussion by eliekh05** [Homebrew/discussions#2872] [Tap maintenance and Homebrew development] [answered]
i have command that is running every time when i want to install for example git 

i type brew install git 

it will say running brew update —preinstall then it will install package for example git  every time this happens and i have to go brew.sh copy the command and paste it and run it every time when this command appear  brew update —preinstall every time can you fix it

**Comment by SMillerDev** [comment, marked as answer]
You don't have to do anything, it's just telling you what it's doing.

**Reply by eliekh05** [reply]
ok
